### DISCUSSION TITLE
文本行检测模型，有这样一个问题，某个显示大小下，能检测成整行，太大了会被错误检测分割成多段，这种除了合并外，有没有其它判断方法，让其以最佳大小进行检测？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12835] [Q&A]
#### 运行环境 / Runtime Environment
- OS:
- Paddle: 2.3
- PaddleOCR: 2.7.5

#### 附件 / Appendix

文本行检测模型，有这样一个问题，某个显示大小下，能检测成整行，太大了会被错误检测分割成多段，这种除了合并外，有没有其它判断方法，让其以最佳大小进行检测？

**Comment by GreatV** [comment]
试试 main 分支下的 slice 操作

**Comment by nissansz** [comment]
怎么操作？

**Comment by nissansz** [comment]
倾斜的文本用哪个模型检测？
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/41010669/dc2f2640-4720-4c3d-ae4b-d3803da84fb4)
